### DISCUSSION TITLE
Community feedback and 2022 roadmap

### DISCUSSION THREAD

**Discussion by Edu4rdSHL** [Findomain/Findomain#161] [Ideas]
Hello everyone, I'm Eduard, the creator of Findomain. First I want to wish everyone and their families a happy year, now let's get straight to the point of this discussion.

The reason why I decided to create it is that despite the fact that Findomain has become known worldwide, there is not much feedback from the community, it is true that the project already has almost everything you can imagine and even more than that it should, perhaps it is because of that complexity, added to the lack of documentation that the community does not fully understand how the tool works and therefore it is difficult to give ideas/suggestions and even discover bugs.

The things that I have in mind and with the highest priority are:

- Create full and detailed documentation for the tool, including examples, use cases and probably videos.
- Add support for other databases than PostgreSQL.
- Create an easy way to import data from external tools without the need to add them directly in the code [as we actually do](https://github.com/Findomain/Findomain/blob/71c9c013f590f12a61f7d8cc05f6ffedb6fa1600/src/external_subs.rs#L1), probably using some kind of templates/plugins.

...and that's all. If you have any idea, question, suggestion or want to say something related to the tool, feel free to leave a comment and we will reply to you for sure.

Again, happy new year and happy hacking.

Ed

**Comment by Edu4rdSHL** [comment]
- https://github.com/Findomain/Findomain/issues/68 will be included in the previous list too.

**Comment by MaxArthurQueen** [comment]
Hi everyone! 
A little tip I discovered thanks to Eduardo: i have very big lists of IPs and domain names to check, and adding this little part: --rate-limit 0 at the end of my cmd reduced by 3/4 the total time it takes yo execute them all. I go very fast since!

The only thing I'm looking for now, so i can be as complete as I can is adding my Facebook n Virustotal tokens API, but I ve got a ZSH problem each time I am trying. I tried to install and update BASH and remove ZSH but still...

Anyone got an idea ?
